angularjs/RichTextEditor/ImportAndExport.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Import and Export in RichTextEditor widget
+title: Import and Export in Angular JS RichTextEditor widget | Syncfusion 
```

Comment:
Angular JS - Don't give space inbetween.
Give like this - AngularJS

Reply:
Done